src/frontend/src/lib/components/core/MenuAddresses.svelte

```diff
@@ -23,6 +25,8 @@
 	<EthWalletAddress />
 {:else if $networkBitcoin}
 	<BtcWalletAddress />
+{:else if $networkSolana}
+	<SolWalletAddress />
 {:else if $pseudoNetworkChainFusion}
 	<WalletAddresses on:icReceiveTriggered={click} />
```

Comment:
It feels as if there should be a uniform way of getting the address to use with a token, so you don't need 4 different address components.  Maybe another thing we can address with support from the backend API.